common/lib/client/auth.ts

```diff
@@ -825,7 +826,7 @@ class Auth {
 				return;
 			}
 			/* RSA4b1 -- if we have a server time offset set already, we can
-			 * autoremove expired tokens. Else just use the cached token. If it is
+			 * auto remove expired tokens. Else just use the cached token. If it is
```

Comment:
I don't think this is correct either. How about "automatically remove"?

Reply:
I think grammatically it would be 'auto-remove' but 'automatically remove' is definitely clearer.